letta/schemas/agent.py

```diff
@@ -21,6 +22,15 @@ class BaseAgent(LettaBase, validate_assignment=True):
     user_id: Optional[str] = Field(None, description="The user id of the agent.")
 
 
+class AgentType(str, Enum):
+    """
+    Enum to represent the type of agent.
+    """
+
+    base_agent = "base_agent"
```

Comment:
I think this would be better named `memgpt_agent`, since `base_agent` might refer to a common base agent class

Reply:
Fixed!